package.json

```diff
@@ -3,23 +3,23 @@
   "description": "Automated dependency updates. Flexible so you don't need to be.",
   "version": "0.0.0-semantic-release",
   "bin": {
-    "renovate": "lib/renovate.js",
+    "renovate": "dist/renovate.js",
```

Comment:
May we should use [ts-node](https://www.npmjs.com/package/ts-node) here, so we don't need to precompile